src/main/java/nl/hsac/fitnesse/util/iban/BEIban.java

```diff
@@ -0,0 +1,232 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+/**
+ * Generates a Belgium IBAN.
+ */
+
+public class BEIban {
+    private RandomUtil randomUtil = new RandomUtil();
+    private IbanUtil ibanUtil = new IbanUtil();
+
+    /**
+     * Generates random number to create IBAN.
+     *
+     * @return random Belgian IBAN.
+     */
+
+    public String generateBEIban(String country, String bankCode) {
+
+        if (country.equals("")) {
```

Comment:
To be honest, I am not to happy with the country classes myself and will refactor to have a generic generator class and a country configuration file for the country specific iban setup as described in this wikipedia article: https://nl.wikipedia.org/wiki/International_Bank_Account_Number

I think it is also better to put the bankcode lists in flat text files and implement a "if bankcode_XX exists, use that" so that we can drop them in later without having to recode anything.